src/main/java/org/example/bookstore/dto/BookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package org.example.bookstore.dto;
 
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
```

Comment:
```suggestion
    @ISBN
```

Reply:
> 

please clarify what i need to change here

Reply:
better to ask those questions on the platform or QnA next time, because we rarely check on you questions here before the actual review